src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/integration/health/HealthCheckTest.kt

```diff
@@ -32,6 +39,24 @@ class HealthCheckTest : IntegrationTestBase() {
       )
   }
 
+  @Test
+  fun `Prison API health reports UP and OK`() {
+    webTestClient.get().uri("/health")
```

Comment:
i am not sure whether we are supposed to use the ping endpoint instead - that is what it says to use for Node apps "to prevent reporting cascading failures": https://dsdmoj.atlassian.net/wiki/spaces/NDSS/pages/2136571940/Node+application+best+practices

Reply:
Although this test calls the health endpoint the call to prison api is /health/ping